config/initializers/capybara.rb

```diff
@@ -0,0 +1,6 @@
+include Capybara::DSL

```

Comment:
Try moving it to each job that needs the Capybara DSL, as here its methods are being over-exposed to the global namespace.

```ruby
class ScraperJob < ActiveJob::Base
  include Capybara::DSL # Add to each Job
  ...
end
```